src/components/Profile/LikedMovies.js

```diff
@@ -0,0 +1,52 @@
+import React, { Fragment } from 'react';
+import { Link, useLocation } from 'react-router-dom';
+import { ChevronLeft } from '../styleElements/icons';
+import { ChevronRight } from '../styleElements/icons';
+import { CirclesBackground } from '../styleElements/CirclesBackground';
+import { dummyData } from '../MatchPage/MatchCard/dummyData';
+import { LogoActive } from '../styleElements/icons';
+
+export function LikedMovies() {
+	const location = useLocation();
+
+	const likedFilmsAll = [];
+	for (let i = 0; i < 8; i++) {
+		likedFilmsAll.push(dummyData[i]);
+	}
+
+	return (
+		<div className='liked_movies__container'>
+			<CirclesBackground />
+
+			<div className='profile__edit-footer'>
+				<div className='profile__chevron-left'>
+					<Link to='/Profile'>
+						<ChevronLeft size={30} active={location.pathname === '/Profile'} />
```

Comment:
This is actually really clever how you did this, I might steal this code snippet for my own purposes.  The only thing is: and this might be totally my fault I can't remember if I had this differently before and changed it out from under you.  But I needed to use a specific color on the ChevronLeft icon so I made it accept "className" instead of "active"  I kinda regret how I coded this icons, they're not super organized and that's on me.

But: if you use this code:

className={location.pathname === '/Profile' ? "activeLogo" : null}

it should achieve what you're trying to do (make the Chevron go from light grey to bright white when you are at '/Profile')